framework/src/multiapps/MultiApp.C

```diff
@@ -842,12 +844,13 @@ MultiApp::createApp(unsigned int i, Real start_time)
 
   _console << COLOR_CYAN << "Creating MultiApp " << name() << " of type " << _app_type
            << " of level " << _app.multiAppLevel() + 1 << " and number " << _first_local_app + i
-           << ":" << COLOR_DEFAULT << std::endl;
+           << " on processor " << processor_id() << " with full name " << full_name << COLOR_DEFAULT
```

Comment:
Not too bad after the cleanups I made at few other places. Each subapp will appear as one line here. Since creating subapp is a little slow, adding this printout can make the screen look alive.

Reply:
With 10k subapps this will suck.  No need to see this.  Trust me - you'll get used to not seeing it.